src/Application/AbstractMagpieCommand.php

```diff
@@ -2,26 +2,41 @@
 
 namespace Ravenfire\Magpie\Application;
 
+use Ravenfire\Magpie\Application\SqlScripts\SqlTraits;
 use Ravenfire\Magpie\Magpie;
 use Symfony\Bridge\Monolog\Handler\ConsoleHandler;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
+/**
+ * Lays foundation for the command files.
+ */
 class AbstractMagpieCommand extends Command
```

Comment:
I just realized an oopsie on my end. `AbstractMagpieCommand` isn't actually abstract and doesn't need to be. There is nothing that something extending this class needs to implement.

So, could you either rename this to just `MagpieCommand` or add `abstract` to the beginning? Or, if you don't want to do it here, we can add that checkbox to another issue.